Jenkinsfile

```
@@ -0,0 +1,193 @@
+pipeline {
+
+  environment {
+    devRegistry = 'ghcr.io/datakaveri/rs-proxy-dev'
+    deplRegistry = 'ghcr.io/datakaveri/rs-proxy-depl'
+    registryUri = 'https://ghcr.io'
+    registryCredential = 'datakaveri-ghcr'
+    GIT_HASH = GIT_COMMIT.take(7)
+  }
+
+  agent { 
+    node {
+      label 'slave1' 
+    }
+  }
+
+  stages {
+    
+    stage('Build images') {
+      steps{
+        script {
+          echo 'Pulled - ' + env.GIT_BRANCH
+          devImage = docker.build( devRegistry, "-f ./docker/dev.dockerfile .")
+          deplImage = docker.build( deplRegistry, "-f ./docker/depl.dockerfile .")
+        }
+      }
+    }
+
+    stage('Unit Tests and Code Coverage Test'){
+      steps{
+        script{
+          sh 'cp /home/ubuntu/configs/rs-proxy-config-test.json ./secrets/all-verticles-configs/config-test.json'
+          catchError(buildResult: 'SUCCESS', stageResult: 'UNSTABLE') {
+              sh "mvn clean test checkstyle:checkstyle pmd:pmd"
+            }  
+        }
+        }
+      post{
+      always {
+        xunit (
+          thresholds: [ skipped(failureThreshold: '1'), failed(failureThreshold: '0') ],
```

Can set skipped failure threshold to 0 
```suggestion
          thresholds: [ skipped(failureThreshold: '0'), failed(failureThreshold: '0') ],
```